stackrox/stackrox
# feat(ui): Minor UX fixes for Workload CVEs

## Description

Various minor UX tweaks - each commit is one change. Descriptions in the testing steps below.

## Checklist

If any of these don't apply, please comment below.

## Testing Performed

1. Adds a copy icon next to image names in workload CVE tables. Clicking the icon copies the full registry + name + tag to the clipboard.
![image](https://github.com/stackrox/stackrox/assets/1292638/705dac63-c593-4e08-87bb-9c64f6d5374f)
![image](https://github.com/stackrox/stackrox/assets/1292638/4f9f709b-e549-4a1d-adaa-a09f330cc637)
_copies: quay.io/stackrox-io/scanner-db:2.31.x-63-gb75b87d225_

2. Change the initial filter category to "Image" instead of "CVE" throughout relevant pages
![image](https://github.com/stackrox/stackrox/assets/1292638/568c6d6b-8f36-419e-9ce7-a269831b0196)
![image](https://github.com/stackrox/stackrox/assets/1292638/2a4d0c8e-478a-4d98-9dfa-b006b963a73c)

3. Add an alert fallback when a description is not found for a CVE. This is part of the issue described by https://issues.redhat.com/browse/ROX-21143, _but no longer occurs locally or on staging_. It is possible this is due to a backend change, or due to CVE data changing over time. The screenshot below uses simulated data to test the effect:
![image](https://github.com/stackrox/stackrox/assets/1292638/ca362939-a817-4121-937c-e2c361394d45)
![image](https://github.com/stackrox/stackrox/assets/1292638/6a4689f5-d15e-433e-8701-f4fdf6f0a1ce)
![image](https://github.com/stackrox/stackrox/assets/1292638/eb9d6042-8053-408e-a55d-16b905534d7b)
